galley/pkg/config/analysis/analyzers/auth/mtls.go

```diff
@@ -82,30 +80,17 @@ func (s *MTLSAnalyzer) Analyze(c analysis.Context) {
 	// TODO Reuse pilot logic as a library rather than reproducing its logic
 	// here.
 
+	mc := util.MeshConfig(c)
```

Comment:
Why is this a utility function rather than a func on `analysis.Context`? It feels more explicit then that every analyzer should be running on exactly one control-plane/mesh configuration.

Reply:
I did fiddle with it briefly but didn't like how it was going, so I was thinking of settling for a smaller incremental improvement. But I agree, it does feel like it should probably end up there.